### ISSUE TITLE
make mail-button configurable

### ISSUE THREAD

**Issue by frontendschlampe** [heiseonline/shariff#41]
We're working on a Contao CMS plugin of Shariff https://github.com/hofff/contao-shariff. Actually the function of the mail-button is a redirect to a recommendation form customized for your website. That's not so good because we can't individualize this function.

A handy solution would be to make it configurable like the twitter-via option, so we can overwrite the redirect target for the page containing the recommendation form.

**Comment by criz** [comment]
+1
Currently this button doesn't make any sense outside of the heise website.

**Comment by pmb0** [comment]
Just released v1.8.0. You can now attach a `data-mail-url` attribute to address an own link. Example:

```html
<div class="shariff" data-mail-url="mailto:foo@example.com"></div>
```
